orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisQueue.kt

```diff
@@ -232,4 +250,13 @@ class RedisQueue(
       tx.block()
       tx.exec()
     }
+
+  private fun JedisCommands.hgetInt(key: String, field: String, default: Int = 0) =
+    hget(key, field)?.toInt() ?: default
+
+  private fun Message.hash() =
+    Hashing
+      .murmur3_32()
+      .hashString(toString(), Charset.defaultCharset())
+      .toString()
```

Comment:
Added a simple hash method. Could, of course, just use the `toString` value itself but this is simple enough.